hadoop-plugin/src/main/groovy/com/linkedin/gradle/zip/HadoopZipExtension.groovy

```diff
@@ -172,6 +173,10 @@ class HadoopZipExtension {
       project.tasks["buildHadoopZips"].dependsOn task;
       dependsOn "startHadoopZips";
 
+      doFirst {
+        mergeTempPropsWithFlowFiles(task, zipName);
```

Comment:
Is there any test for merging the tempprops with flow YAML files in the zip task?